src/benchmarks/micro/libraries/System.Collections/Add/TryAddDefaultSize.cs

```diff
@@ -33,6 +33,26 @@ public class TryAddDefaultSize<T>
                 collection.TryAdd(uniqueValues[i], uniqueValues[i]);
             return collection;
         }
+
+        [Benchmark]
+        public SortedDictionary<T, T> SortedDictionary()
+        {
+            var collection = new SortedDictionary<T, T>();
+            var uniqueValues = _uniqueValues;
+            for (int i = 0; i < uniqueValues.Length; i++)
+                collection.TryAdd(uniqueValues[i], uniqueValues[i]);
+            return collection;
+        }
+
+        [Benchmark]
+        public SortedList<T, T> SortedList()
+        {
+            var collection = new SortedList<T, T>();
+            var uniqueValues = _uniqueValues;
+            for (int i = 0; i < uniqueValues.Length; i++)
+                collection.TryAdd(uniqueValues[i], uniqueValues[i]);
+            return collection;
+        }
```

Comment:
I didn't end up adding TryAdd benchmarks for the immutable collections here because they just use the TryAdd extension which does not work for them.